tests/template_tests/test_response.py

```diff
@@ -221,6 +221,12 @@ def test_pickling_cookie(self):
 
         self.assertEqual(unpickled_response.cookies['key'].value, 'value')
 
+    @override_settings(ROOT_URLCONF='template_tests.urls',
```

Comment:
`template_tests` isn't the correct place for this test (it has nothing to do with the template system). Please look around for  a better place (if you need help or want to confirm before moving things, let me know).